### PR TITLE
Extend trenzn

### PR DESCRIPTION
**Pull request by BenjaminBrandmeier**
This PR extends the meaning of trenzn. 

Additionally a real-world example sentence including the word trenzn is included.

### CODE REVIEW

File: bayrischer_schmaaz.csv [hunk 1]
```diff
@@ -344,7 +344,7 @@ sobbn,"langsam, schleichend gehen",
 soifen,sabbern,
 Suri,Rausch,
 tramhabbad,verträumt,
-trenzn,"weinen, schluchzen",
+trenzn,"weinen, schluchzen oder etwas verschütten",Hosd di scho wida o'trenzd du Fagge!
```
Comment (anetz89) [inline bayrischer_schmaaz.csv]: please add a comma in front of 'oder'

**Review by anetz89 — CHANGES_REQUESTED** [review]
overall a great commit, but I would have expected correct grammar

File: bayrischer_schmaaz.csv (hunk 1 above)
Comment (BenjaminBrandmeier) [reply bayrischer_schmaaz.csv]: Absolutely!

**Review by anetz89 — APPROVED** [review]
soidad jetz bassn ;)
wobei zwoa zeilen vll schena warn...

**Comment by anetz89** [comment]
weils wuaschd is

**Pull request merged**